code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -474,9 +474,11 @@ emp_act
 			return FALSE
 
 	var/obj/item/organ/external/affecting = get_organ(ran_zone(user.zone_selected))
+
+	// if the targeted limb doesn't exist, pick a new one at random so you don't have to swap target zone
 	if(!affecting)
-		to_chat(user, "<span class='danger'>They are missing that limb!</span>")
-		return FALSE
+		affecting = pick(bodyparts)
+
```

Comment:
You might want this BEFORE ran_zone is called, as that returns a random zone if it fails a prob(80) check. IMO it makes more sense to pick one if youre not targetting, but you can still miss if you get unlucky with ran_zone (probably best to still remove the "they are missing that limb" in this case

Reply:
Contrabang and I looked at this a few times over the last week, and they eventually suggested to just keep it how it is after the alternatives had issues.